client/my-sites/marketing/tools/index.tsx

```diff
@@ -66,7 +55,7 @@ export const MarketingTools: FunctionComponent< MarketingToolsProps > = ( {
 				<MarketingToolFeature
 					title={ translate( 'Need an expert to help realize your vision? Hire one!' ) }
 					description={ translate(
-						"We’ve partnered with Upwork, a network of freelancers with a huge pool of WordPress experts. Hire a pro to help build your dream site."
+						'We’ve partnered with Upwork, a network of freelancers with a huge pool of WordPress experts. Hire a pro to help build your dream site.'
```

Comment:
Sorry, that's an issue of my making: I've copy/pasted these strings from the P2 which automatically replaced those quotes. I indeed think we should convert every `’` in to `'` as I suspect it's just easier for translators to use the latter (frankly I have no idea how to type those curly quotes with my keyboard).